language/compiler/ir_to_bytecode/syntax/src/ast.rs

```diff
@@ -281,18 +302,20 @@ pub enum FunctionCall {
     ModuleFunctionCall {
         module: ModuleName,
         name: FunctionName,
+        type_parameters: Vec<Type>,
     },
 }
 /// The type for a function call and its location
 pub type FunctionCall_ = Spanned<FunctionCall>;
 
 /// Enum for Move commands
+#[allow(clippy::large-enum-variant)]
```

Comment:
I'm a bit unsure if this is the right thing to do. My thoughts:
  1) do we have a custom threshold value that triggers this check?
  2) if 1) is not true, then it means the size is over 200, which is bad.
  3) if 1) is true and we have a small threshold, then this may be fine.